cmd/manager/main.go

```diff
@@ -20,7 +20,7 @@ import (
 	"os"
 
 	"github.com/kubeflow/xgboost-operator/pkg/apis"
-	"github.com/kubeflow/xgboost-operator/pkg/controller"
+	controller "github.com/kubeflow/xgboost-operator/pkg/controller/v1"
```

Comment:
This means the master will point to v1 changes. This is expected, right?

Reply:
Yes. This is intended so more users would start testing v1 by default.